src/runtime_src/core/common/drv/cu_hls.c

```diff
@@ -122,13 +122,89 @@ static void cu_hls_check(void *core, struct xcu_status *status)
 	status->num_ready = ready_reg;
 }
 
+static void cu_hls_enable_intr(void *core, u32 intr_type)
+{
+	struct xrt_cu_hls *cu_hls = core;
+	u32 *base_addr = cu_hls->vaddr;
```

Comment:
Please use the same pointer type in cu_hls_enable_intr() and cu_hls_disable_intr().